src/Common/Throttler.h

```diff
@@ -97,6 +102,19 @@ class Throttler
 
         count = 0;
         watch.reset();
+        accumulated_sleep = 0;
+    }
+
+    /// Is throttler already accumulated some sleep time and throttling.
+    bool isThrottling() const
```

Comment:
For me this method is unneeded. It is just like `size()` method of some concurrent data structure. This method won't tell you any needful information. And knowledge about throttling goes into other classes, so we have stronger dependencies.